modules/ROOT/pages/api-led-design.adoc

```diff
@@ -123,45 +123,44 @@ image:get-button.png[GET button, 500]
 . Click *GET* to display the GET request and more information about the specification.
 +
 image:code-response1.png[Response field, 500]
-. Click *Code examples* to review samples for each protocol.
-. Click *200* and *404* under *Responses* to review the responses defined in the API specification.
-. Click the blue *Try it* button.
+. Click *Show* to review code examples for each protocol.
+. Click the *200* and *404* tabs to review the responses defined in the API specification.
+. Click the *Try it* button.
 +
 image:try-it.png[The Try It button, 500]
 . Click *Send* to send your request to the temporary request URL that the mocking service creates from your specification.
 +
-image:ignore-this.png[Error messages you can ignore, 500]
-+
 It's safe to ignore any error messages on this screen. A successful request returns `200 OK` and the test message:
 +
 image:successful-test1.png[Results of a successful test, 500]
-. Click *Response details* in the kebab menu to examine the response headers and request headers in the mocking service to help diagnose issues or understand the behavior of your API specification.
-. When you have finished testing, open the *Mocking Service Configuration* panel, then in *Local Settings* enable *Select By Default*.
+. Select *Response details* in the more menu to examine the response headers and request headers in the mocking service to help diagnose issues or understand the behavior of your API specification.
```

Comment:
If "more menu" is a kebab...
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-ellipsismenu